project_5.html

```diff
@@ -607,8 +606,7 @@ <h2 class="accordion-header">
 					<!--Pin to page?-->
 					<div id="schemCollapse2" class="accordion-collapse collapse" data-bs-parent="#schemAccordian">
 						<div class="accordion-body text-center">
-							<img src='assets/images/project_5/ops23_project5_schemB.png' class="pj-schem"
-								style="max-width: 500px;">
+							<img class="pj-img-fluid mx-auto" src='assets/images/project_5/ops23_project5_schemB.png' width="400px">
```

Comment:
![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/75911565/9d857c67-c335-427f-b941-ca94f3ba4b3d)

this image is off on mobile

![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/75911565/63ab0e19-383e-486f-a4e8-2dbbf21bce37)